### PR TITLE
Add gemspec for use as gem based theme

### PR DESCRIPTION
**Pull request by wbreeze**
This adds `jekyll-theme-feeling-responsive.gemspec` for building a gem and pushing to [RubyGems](https://rubygems.org/gems/jekyll-theme-feeling-responsive).

### CODE REVIEW

File: jekyll-theme-feeling-responsive.gemspec [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Gem::Specification.new do |s|
+  s.name = 'jekyll-theme-feeling-responsive'
+  s.version = '1.0.2'
```
Comment (TWiStErRob) [inline jekyll-theme-feeling-responsive.gemspec:3]: 1.9.1 https://phlow.github.io/feeling-responsive/changelog/

File: Gemfile.lock [hunk 2]
```diff
@@ -0,0 +1,53 @@
+GEM
```
Comment (TWiStErRob) [inline Gemfile.lock:1]: I don't think this file should be added. `Gemfile` already has the dependency list.

File: jekyll-theme-feeling-responsive.gemspec (hunk 1 above)
Comment (wbreeze) [reply jekyll-theme-feeling-responsive.gemspec:3]: Hi! Thank you for the review.  Are you suggesting an update to the change log?  I searched the referenced link for mention of this feature.  Has it already been done?

The change log is for the `gh-pages` branch.  It doesn't exist on the `bare-bones-version` branch.  This change is to the `bare-bones-version` branch.  Do you think the change belongs on the `gh-pages` branch?

File: Gemfile.lock (hunk 2 above)
Comment (wbreeze) [reply Gemfile.lock:1]: Hey, thank you again for reviewing!

In Ruby land it's very standard to check-in the Gemfile.lock.  And in fact the bundler documentation [suggests it](http://bundler.io/v1.15/guides/bundler_sharing.html).

The reason is that `Gemfile.lock` captures the exact versions of the specified gems and of the dependent gems that are pulled-in transitively.  It allows anyone checking-out the repository to build with the same versions of every gem, exactly matching those in use by the person who checked-in or pushed.

However I won't die on this hill! If you want it removed to approve, I'll make it so.

File: jekyll-theme-feeling-responsive.gemspec (hunk 1 above)
Comment (TWiStErRob) [reply jekyll-theme-feeling-responsive.gemspec:3]: Ah, then probably it's correct, @Phlow can always update.
I meant to not diverge release and gem version.

File: Gemfile.lock (hunk 2 above)
Comment (TWiStErRob) [reply Gemfile.lock:1]: Interesting, thanks for the link.

Comment (wbreeze) [reply Gemfile.lock:1]: Thank you for looking at the PR's!

**Comment by Phlow** [comment]
Thankyou @wbreeze and @TWiStErRob

**Comment by TWiStErRob** [comment]
@Phlow I just noticed something, if this is the bare bones, then shouldn't the gem name reflect that?

**Comment by Phlow** [comment]
Yes. You're right.

Must I change this here › `s.name = 'jekyll-theme-feeling-responsive'`

**Comment by TWiStErRob** [comment]
think so, and rename the file

**Comment by wbreeze** [comment]
The gems I think are already bare bones, usually.  Possibly, with the gem available, there's no need to maintain two branches (with the need to move changes to one downstream into the other).

The files in the gem are controlled in the gemspec; so, the gem could live safely in gh-pages without need for a separate bare bones branch.

However that is to your taste, as you like it. Just an idea. And perhaps in hindsight I should have done it that way- put the gem on the gh-pages branch.

**Comment by Phlow** [comment]
@wbreeze I know, but I currently have no time and no energy to maintain it. It costs to many hours of work. I prefer to spend this time with my kids ;)

**Comment by Phlow** [comment]
I even was thinking about deleting the whole repo so people are not dissappointed, that i don't maintain it anymore.

**Comment by wbreeze** [comment]
People would be disappointed if you deleted it because they might be using it.  It looks really great. 
 There's always the add collaborators option in the project settings.  Yes, by all means, time with kids.  They grow up fast.

**Comment by TWiStErRob** [comment]
Maintenance doesn't necessarily mean actively developing it, it's pretty stable, and you're accepting PRs. I think that's plenty enough. Maybe put up a notice in README that PRs are preferred over asking for a feature in issues.

**Comment by Phlow** [comment]
@TWiStErRob Thankyou for your thoughts. I agree. I don't mind accepting pull requests. Unless they just make this already complex theme even more complicated.

**Comment by wbreeze** [comment]
@Phlow if you would like me to make you an owner of [the gem on rubygems](https://rubygems.org/gems/jekyll-theme-feeling-responsive) I will use the [gem owner command](http://guides.rubygems.org/command-reference/#gem-owner) to make it so.  Mostly I won't be keeping it updated. All I need to do that is the email address you want used for rubygems.org.

**Pull request merged**